authapi/views.py

```diff
@@ -158,16 +158,52 @@ class TeamPermissionViewSet(
     '''Nested viewset to add and remove permissions from teams.'''
     queryset = SeedPermission.objects.all()
     serializer_class = PermissionSerializer
+    permission_classes = (permissions.TeamPermissionPermission,)
 
-    def create(self, request, parent_lookup_seedteam=None, **kwargs):
+    def check_team_permissions(self, request, teamid, orgid=None):
+        old_method = request.method
+        request.method = 'GET'
```

Comment:
Resolved in https://github.com/praekelt/seed-auth-api/pull/28/commits/8ffd467a809a37306a8971c9f76fe415f0bf1402